### PR TITLE
make `get_tag` and `tags` public

### LINKED ISSUE #69: Other tags?
I see that both `get_tag` and `tags` are not public, is there any reason for this? I'd love to be able to query what other tags may be in the header.

### PR DESCRIPTION
**Pull request by int0xh**
closes #69

### CODE REVIEW

**Comment by strcoy** [comment]
@IsaacWoods pinging for review! :smile:

**Comment by IsaacWoods** [comment]
Hi @strcoy

Out of interest, is the crate lacking support for a tag you need, or is this just for maximal permissiveness?

**Comment by phip1611** [comment]
@IsaacWoods Should we add this to the v1.0.0 release?  This would fit because we also have to make the tag type enum public.

File: src/lib.rs [hunk 1]
```diff
@@ -227,11 +227,13 @@ impl BootInformation {
         unsafe { &*self.inner }
     }
 
-    fn get_tag<'a>(&'a self, typ: u32) -> Option<&'a Tag> {
+    /// Search for a tag by `type` identifier.
+    pub fn get_tag<'a>(&'a self, typ: u32) -> Option<&'a Tag> {
```
Comment (IsaacWoods) [inline src/lib.rs:231]: We don't use `get_*` for anything else - I think we should use this as an opportunity to just rename this to `tag`.

**Comment by phip1611** [comment]
As we have separate getters for every single possible tag, which returns Options, I don't see why this should be public. This would be more confusing than benefitical. I'd propose that we check if we have indeed a getter for every single tag type and if not, we do this instead in this PR. Opinion, @IsaacWoods ?

**Comment by IsaacWoods** [comment]
I'm inclined to agree - if a tag type is missing, the preferred answer imo is to add it to this library. With a format like `stivale`, where I think custom tag types are allowed, this would make more sense, but yeah I'm inclined to close unless the original author comes back to us.

**Comment by int0xh** [comment]
> I'm inclined to agree - if a tag type is missing, the preferred answer imo is to add it to this library. With a format like `stivale`, where I think custom tag types are allowed, this would make more sense, but yeah I'm inclined to close unless the original author comes back to us.

Makes sense! Closing it.

**Pull request closed without merging**